masterlist.yaml

```diff
@@ -32,7 +32,7 @@ common:
     type: warn
     content:
       - lang: en
-        text: "You seem to be using %1%, but you haven't enabled the %1% patch for this plugin: %2%"
+        text: "You seem to be using %1%, but you haven't enabled the patch for this plugin: %2%"
```

Comment:
Not sure about this, but I think its easier to read.

> You seem to be using True Storms SE, but you haven't enabled the patch for this plugin: NAT - TrueStorms Merged Compatibility SSE

> You seem to be using True Storms SE, but you haven't enabled the True Storms SE patch for this plugin: NAT - TrueStorms Merged Compatibility SSE

Reply:
I think the new wording is better, and I like "You seem to be using" just in case they're not actually using that mod, but another one with the same plugin name (which happens). Better to sound a little unsure than to say "You are using X" and have the user respond with "uh, no I'm not".